pkg/ipam/option/option.go

```diff
@@ -36,3 +36,5 @@ const (
 	IPAMDoNotRelease    = "do-not-release"
 	IPAMReleased        = "released"
 )
+
+const ENIPDBlockSizeIPv4 = 16
```

Comment:
Please add a doc comment stating that this is the number of IPs in a ENI PD and that the prefix size is hardcoded to `/28`.